src/purify.js

```diff
@@ -947,7 +949,7 @@ function createDOMPurify(window = getGlobal()) {
         }
 
         arrayPop(DOMPurify.removed);
-      } catch (error) {}
+      } catch (_) {}
```

Comment:
This is ESLint fixes. Cause we don't use the arg to indicate that better.

Reply:
Yeah, had to do the same a while ago